Sources/ApolloWebSocket/DefaultImplementation/SSLSecurity.swift

```diff
@@ -205,8 +203,8 @@ open class SSLSecurity : SSLTrustValidator {
         SecTrustCreateWithCertificates(cert, policy, &possibleTrust)
 
         guard let trust = possibleTrust else { return nil }
-        var result: SecTrustResultType = .unspecified
-        SecTrustEvaluate(trust, &result)
+        _ = SecTrustEvaluateWithError(trust, nil)
```

Comment:
The result was not being used here so we don't store it either with the new call. Again, no error was being reported perviously.

_`SecTrustEvaluateWithError(:,:)` is the synchronous replacement of the deprecated `SecTrustEvaluate(:,:)`. It is now a simple boolean return._